appcheck/app/build.gradle

```diff
@@ -0,0 +1,48 @@
+plugins {
+    id 'com.android.application'
+}
+apply plugin: 'kotlin-android'
```

Comment:
nit: use same approach to apply plugins. `apply plugin: ...` for consistency with other plugins.